src/api/entitlements.js

```diff
@@ -294,8 +294,15 @@ export class Entitlement {
    * @param {!Array<string>} products
    * @param {string} subscriptionToken
    * @param {JsonObject|null|undefined} subscriptionTokenContents
+   * @param {Timestamp} subscriptionTimestamp
    */
-  constructor(source, products, subscriptionToken, subscriptionTokenContents) {
+  constructor(
```

Comment:
Adding this to the entitlement object feels wrong - can we do a VC to discuss

Reply:
Sure, this PR is still in progress. But we can discuss about that.